certified-connectors/Encodian/apiDefinition.swagger.json

```diff
@@ -15579,172 +16930,672 @@
           "x-ms-summary": "Privileges",
           "description": "Set the privileges assigned to the PDF document"
         },
-        "cryptoAlgorithm": {
-          "default": "AESx256",
-          "enum": [
-            "RC4x40",
-            "RC4x128",
-            "AESx128",
-            "AESx256"
-          ],
-          "type": "string",
-          "x-ms-visibility": "advanced",
-          "x-ms-summary": "Cryptographic Algorithm",
-          "description": "Set the cryptographic algorithm"
-        },
+        "cryptoAlgorithm": {
+          "default": "AESx256",
+          "enum": [
+            "RC4x40",
+            "RC4x128",
+            "AESx128",
+            "AESx256"
+          ],
+          "type": "string",
+          "x-ms-visibility": "advanced",
+          "x-ms-summary": "Cryptographic Algorithm",
+          "description": "Set the cryptographic algorithm"
+        },
+        "pdfPrivilegesAllowAssembly": {
+          "default": false,
+          "type": "boolean",
+          "x-ms-visibility": "advanced",
+          "x-ms-summary": "Privilege - Assembly",
+          "description": ""
```

Comment:
@Encodian can you please explain why you have empty descriptions? Please define a proper description.
Please fix all empty descriptions in the swagger file.

Reply:
@Amjed-Ayoub - This hasn't changed since the previously approved version (for over a year in fact), we auto-generate the swagger definition in code so it is not a quick fix. We can raise a ticket to address but would request this is approved as it matches the current approved version, is this possible?

Reply:
Resolved with update